core/src/main/java/org/apache/accumulo/core/fate/accumulo/AccumuloStore.java

```diff
@@ -70,12 +71,22 @@ public long create() {
   }
 
   @Override
-  protected List<String> getTransactions() {
-    return scanTx(scanner -> {
+  protected Stream<FateIdStatus> getTransactions() {
+    try {
+      Scanner scanner = context.createScanner(tableName, Authorizations.EMPTY);
       scanner.setRange(new Range());
       TxColumnFamily.STATUS_COLUMN.fetch(scanner);
-      return scanner.stream().map(e -> e.getKey().getRow().toString()).collect(Collectors.toList());
-    });
+      return scanner.stream().onClose(scanner::close).map(e -> {
+        return new FateIdStatus(parseTid(e.getKey().getRow().toString())) {
+          @Override
+          public TStatus getStatus() {
+            return TStatus.valueOf(e.getValue().toString());
+          }
+        };
+      });
```

Comment:
```suggestion
      return scanner.stream().onClose(scanner::close)
          .map(e -> new FateIdStatus(parseTid(e.getKey().getRow().toString())) {
            @Override
            public TStatus getStatus() {
              return TStatus.valueOf(e.getValue().toString());
            }
          });
```

Reply:
Just a small simplification which you can ignore

Reply:
I tried removing the code block when writing and thought it was just too much going on on a single line.  Decided to add the code block back so that the lambda declaration and anonymous class declaration were on separate lines.